devops/business_services/s3_buckets.rst

```diff
@@ -0,0 +1,48 @@
+S3 Buckets
+==========
+
+
+Naming Conventionen for S3
```

Comment:
conventions?

Reply:
I disagree, capitalized because it's in a title. I'm not 100% sure about it being singular, [dictionary](https://en.oxforddictionaries.com/definition/convention) list it as countable and uncountable. I'd probably go with singular.

Reply:
(I don't care about capital letters and plural vs. singular, but "Conventionen" isn't an english word afaik)

Reply:
Oh, guess I didn't read it carefully enough.